c2corg_ui/static/js/alertservice.js

```diff
@@ -26,6 +26,20 @@ app.Alerts = function(gettextCatalog) {
 
 
 /**
+ * Use this function to annotate a string for extraction by the
+ * gettext extract tool. The call to this function will be eliminated
+ * by the Closure compiler when minifying, so it should not have any
+ * performance effect.
+ * See https://angular-gettext.rocketeer.be/dev-guide/annotate-js/
+ * @param {string} str String to have extracted by gettext tool
+ * @return {string}
+ */
+app.Alerts.prototype.gettext = function(str) {
```

Comment:
This function deals with a very specific use case and I think we currently only need it on `app.Alerts`. I would rather keep it there for now and only create the generic version if
we really need it.